src/Frontend-GNOME/Views/Chats/ChatView.cs

```diff
@@ -578,6 +582,14 @@ public virtual void Populate()
             _IsSynced = true;
         }
         
+        public virtual void UpdateLastSeen() {
+            _OutputMessageTextView.UpdateMarkerline();
+            if (_OutputMessageTextView.LastMessage != null) {
```

Comment:
This is racy, use a local var (once you add the background thread, it will be clear why)